src/site/_includes/macros/audio-fab.njk

```diff
@@ -0,0 +1,16 @@
+{% macro audioFab(thumbnail, target="audio-player") %}
+<web-audio-fab target="{{ target }}">
+  <button class="web-audio-fab__button button" data-playing aria-label="Back to audio player">
+    {% Img class="web-audio-fab__thumbnail", src=thumbnail, alt="", width="56", height="56" %}
+    <div class="web-audio-fab__equalizer-container">
+      <svg class="web-audio-fab__equalizer" xmlns="http://www.w3.org/2000/svg" width="13" height="8" viewBox="0 0 13 8">
```

Comment:
No pressure, but do we want to start to handle svgs like we do on dcc, `{% include 'icons/audio-fab.svg' %}`

Reply:
In general yes, and I've been using the icons macro elsewhere in web.dev.

In this case I didn't because I wanted to apply classes to the svg elements themselves and the macro doesn't support that.